OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
I agree, and I think the best part of this OWNERS system is that you can specify people in a specific area, which means we can put more people in the package he/she is mainly working on. The root OWNERS might be more suitable for people taking charge of the whole repo like Kuat and rshriram.